network/benchmarks/netperf/nptest/nptest.go

```diff
@@ -70,13 +70,13 @@ var globalLock sync.Mutex
 const (
 	workerMode           = "worker"
 	orchestratorMode     = "orchestrator"
-	iperf3Path           = "/usr/local/bin/iperf3"
-	netperfPath          = "/usr/local/bin/netperf"
-	netperfServerPath    = "/usr/local/bin/netserver"
+	iperf3Path           = "/usr/bin/iperf3"
+	netperfPath          = "/usr/bin/netperf"
+	netperfServerPath    = "/usr/bin/netserver"
```

Comment:
These paths look like something that should live in a configuration file or perhaps as a defaults for values pulled from configuration file.